### DISCUSSION TITLE
how to access self learning modules and student developer pack!

### DISCUSSION THREAD

**Discussion by akhil9281** [github/india#300] [Students]
the self learning modules aren't working !!
nothing's opening by clicking them.

similarly there's an option to "get your developer pack" but it's not leading to anywhere either! IT's not even opening..
anyone has any idea what to do or how to proceed ??

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
